handbook/company.md

```diff
@@ -360,7 +360,7 @@ These humans might be working within the "Interface" group itself insofar as the
 - Make upgrading Fleet versions seamless for users.
 - Improve the integrity of data (both collected directly from agents or derived from that data, e.g., vulnerabilities).
 - Maintain quick time-til-merge timeframe for PRs reviewed.
-- Maintain clean, empathetic interfaces that allow contributors in other groups (or from outside the company) to execute quickly and without the need to wait for review or approvals
+- Maintain clean, empathetic interfaces that allow contributors in other groups (or from outside the company) to execute quickly and without the need to wait for reviews or approvals
```

Comment:
Let's go with "review or approval" (both singular) to encourage small numbers of reviews (ideally only 1)